src/reducers/sources.js

```diff
@@ -284,8 +284,12 @@ export function getSourceByURL(state: OuterState, url: string) {
   return state.sources.sources.find(source => source.get("url") === url);
 }
 
+export function getSourceInSources(sources: I.Map<string, Source>, id: string) {
```

Comment:
Yeah - the underscore was meh when it was a private function, but below it's public... so i don't love it